zanata-common-api/src/main/java/org/zanata/rest/GenerateSamples.java

```diff
@@ -40,7 +40,7 @@
 
     public GenerateSamples(PrintStream out) {
         mapper = new ObjectMapper();
-        mapper.getSerializationConfig().enable(Feature.INDENT_OUTPUT);
+        mapper.getSerializationConfig().with(Feature.INDENT_OUTPUT);
```

Comment:
This whole class should either be reworked into a unit test, documented as a utility, or else deleted before it costs any more maintenance.  I think Asgeir wrote it, so we'll have to work out whether it has any use just from reading the code.

https://code.google.com/p/flies/source/list?path=/common/zanata-common-api/src/main/java/org/zanata/rest/GenerateSamples.java&r=de6d7ad81b8c051882f9e1586f70174459418ec5